operator/pkg/vfs/assets.gen.go

```diff
@@ -1,247 +1,249 @@
-// ../../data/charts/security/nodeagent/templates/_affinity.tpl
-// ../../data/charts/security/nodeagent/templates/clusterrole.yaml
-// ../../data/charts/security/nodeagent/templates/clusterrolebinding.yaml
-// ../../data/charts/security/nodeagent/templates/daemonset.yaml
-// ../../data/charts/security/nodeagent/templates/serviceaccount.yaml
-// ../../data/charts/security/nodeagent/values.yaml
-// ../../data/examples/googleca/values-istio-google-ca.yaml
-// ../../data/examples/multicluster/values-istio-multicluster-gateways.yaml
-// ../../data/examples/multicluster/values-istio-multicluster-primary.yaml
-// ../../data/examples/user-gateway/ingress-gateway-only.yaml
-// ../../data/examples/vm/values-istio-meshexpansion-gateways.yaml
-// ../../data/examples/vm/values-istio-meshexpansion.yaml
-// ../../data/operator/Chart.yaml
-// ../../data/operator/templates/clusterrole.yaml
-// ../../data/operator/templates/clusterrole_binding.yaml
-// ../../data/operator/templates/crd.yaml
-// ../../data/operator/templates/deployment.yaml
-// ../../data/operator/templates/namespace.yaml
-// ../../data/operator/templates/service.yaml
-// ../../data/operator/templates/service_account.yaml
-// ../../data/profiles/default.yaml
-// ../../data/profiles/demo.yaml
-// ../../data/profiles/empty.yaml
-// ../../data/profiles/minimal.yaml
-// ../../data/profiles/remote.yaml
-// ../../data/profiles/sds.yaml
-// ../../data/translateConfig/reverseTranslateConfig-1.4.yaml
-// ../../data/translateConfig/reverseTranslateConfig-1.5.yaml
-// ../../data/translateConfig/translateConfig-1.3.yaml
-// ../../data/translateConfig/translateConfig-1.4.yaml
-// ../../data/translateConfig/translateConfig-1.5.yaml
-// ../../data/versions.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/Chart.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/files/crd-all.gen.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/files/crd-certmanager-10.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/files/crd-certmanager-11.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/files/crd-mixer.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/kustomization.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/templates/clusterrole.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/templates/clusterrolebinding.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/templates/crds.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/templates/endpoints.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/templates/namespaces.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/templates/serviceaccount.yaml
+// ../../../../../../../../../tmp/istio-charts.SlJN1fTY1C/charts/base/templates/services.yaml
```

Comment:
this looks non-stable?

Reply:
Even the `..` is not stable since different users will have different amounts. So maybe we need to `cd` to the tmp dir and run from there

Reply:
yes, it's not stable - i didn't think it would matter too much. anyway, it's easy enough to fix.

Reply:
done.

Reply:
it matters since everyone running it changes it. but gen check fails too